python/GafferArnoldUI/ArnoldLightFilterUI.py

```diff
@@ -91,6 +100,13 @@ def __activateRoundness( parent ) :
 
 		"parameters.shader" : [
 
+			"description",
+			"""
+			Shader to be used for the light_blocker filter. Only works if geometry type
```

Comment:
I'm seeing the opposite - it always works, but it causes the geometry type to be ignored.

Reply:
Really? If you set the type to "sphere" and plug in a shader, you still see some filtering happening? From the Arnold docs: "It only works when 'Geometry Type' is set to Box."

Reply:
Yeah, it's like the shader just provides the final filter value (example pasted in previous comment).